### DISCUSSION TITLE
my-mac-os-zsh-profile/

### DISCUSSION THREAD

**Discussion by giscus** [natelandau/natelandau.com#62] [Comments]
# my-mac-os-zsh-profile/

I've spent years curating a collection of Mac aliases and shortcuts to make my life easier. My full .zshrc is below, feel free to take whatever you find useful and put it to good use.

https://natelandau.com/my-mac-os-zsh-profile/

**Comment by binarybottle** [comment]
Thank you very much!  I appreciate the thought you've given to constructing an efficient and pleasing terminal experience, and I look forward to using and adapting your settings to my needs.

**Comment by ycktw** [comment]
Thank you for the detailed tutorial. I also directly used the profile from your GitHub.

As an engineer, please allow me to point out a typo:

The line "git checkout https://github.com/natelandau/dotfiles.git" should be "git clone https://github.com/natelandau/dotfiles.git".
